### PR TITLE
LaTeX/style guidelines v2

### PR DESCRIPTION
**Pull request by ezgimez**
hopefully more LaTeX/style guidelines... @yp717  said he would write more :)

### CODE REVIEW

File: LaTeX_guidelines.md [hunk 1]
```diff
@@ -101,5 +100,27 @@ Example:
...
+- and
+- nor
+- but
+- or
+- yet
+- so
+
+Note that "FANBOYS" are used to connect *clauses*. 
+
+10. Do NOT use comma before "and" for the case of connecting words.
```
Comment (yp717) [inline LaTeX_guidelines.md:116]: can you add an 11. here

11. Use full stops appropriately with commas. If the quote can be replaced conceptually by emphasis/bold text, then the full stop should be outside the quotation marks. If the sentence ends within the quote the fullstop should be inside the quote
- e.g. 1: According to Byron, "Our burgers are fantastic."
- e.g. 2: The LaTeX guidelines are "fun".

**Review by yp717 — CHANGES_REQUESTED** [review]
left one comment for an extra point and then can approve

File: LaTeX_guidelines.md (hunk 1 above)
Comment (ezgimez) [reply LaTeX_guidelines.md:116]: done

**Review by yp717 — APPROVED** [review]
lgtm

**Pull request merged**